crates/telio-model/src/features.rs

```diff
@@ -568,22 +586,32 @@ pub struct Features {
     pub multicast: bool,
 }
 
-impl FeaturePaths {
```

Comment:
This was moved to earlier in the file with no other changes to this block of code